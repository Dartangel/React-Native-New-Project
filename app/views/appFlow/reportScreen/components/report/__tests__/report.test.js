import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Report from '../report';

describe('Report', () => {
    test('Report snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Report />);
        expect(snapshot).toMatchSnapshot();
    });
});