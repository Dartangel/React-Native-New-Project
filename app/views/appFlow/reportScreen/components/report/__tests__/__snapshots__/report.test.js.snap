// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report Report snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#0F2635",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.5,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-around",
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#08DBA5",
            "fontSize": 20,
          }
        }
      >
        Object {}
      </Text>
      <Text
        style={
          Object {
            "color": "#08DBA5",
            "fontSize": 20,
          }
        }
      >
        Object {}
      </Text>
      <Text
        style={
          Object {
            "color": "#08DBA5",
            "fontSize": 20,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0.5,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-around",
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;
